recipes/Functions.py

```diff
@@ -82,6 +83,14 @@ def render_form_v2(self):
             height=300,
         )
 
+    def get_price_roundoff(self, state: dict) -> float:
+        try:
+            # called from another workflow don't charge any credits
+            CalledFunction.objects.get(function_run=self.get_current_sr())
```

Comment:
cant use .get() here because multiple objects may exist here and it will throw MultipleObjectsReturned error